ethereumj-core/src/main/java/org/ethereum/publish/exprimental/EventBus.java

```diff
@@ -0,0 +1,55 @@
+package org.ethereum.publish.exprimental;
+
+import org.ethereum.publish.event.Event;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.function.Consumer;
+
+import static java.util.Objects.isNull;
+
+public class EventBus {
```

Comment:
Maybe it is better to move `experimental` stuff to a separate branch?

Reply:
I'll remove this package completely.